source/main/gui/panels/GUI_MainSelector.cpp

```diff
@@ -67,6 +67,11 @@ void MainSelector::Show(LoaderType type, std::string const& filter_guid)
     {
         m_selected_entry = m_last_selected_entry[m_loader_type];
     }
+
+    if (type == LT_Skin)
+    {
+         m_selected_entry = m_selected_entry["Default"];
```

Comment:
😱 I had**NO** idea this could even compile, but apparently it does: http://madebyevan.com/obscure-cpp-features/